scipy/stats/_distn_infrastructure.py

```diff
@@ -2212,21 +2239,36 @@ def fit(self, data, *args, **kwds):
               function to be optimized) and ``disp=0`` to suppress
               output as keyword arguments.
 
+            - method : The method to use. The default is "MLE" (Maximum
+              Likelihood Estimate); "GMM" (Generalized Method of Moments)
+              is also available.
```

Comment:
What about cases when method  "GMM" and "MLE" provide the same results?
would a route to MLE make sense or would a separate method still be used?

Reply:
I don't really understand. Are there distributions for which it is guaranteed that the MLE parameter estimate will be identical to the GMM parameter estimate for any input data? If so, I did not consider this. I suppose for those distributions the `fit` method should be overridden, and it can decide which method to use - presumably whichever is more efficient.

Does that answer your question?

Reply:
@mdhaber I somehow edited your comment rather than quote the reply, and it seems to have overwritten your response and I cannot seem to get it back from my browser cache, apologies. You were saying that if the two are the same, then whichever is faster would make sense to use and it would also make sense for the user to override the `.fit` in that case. Which seems reasonable here.

Reply:
I suppose someone could write a script that tests stuff like this - whether MLE and MM give the same result, whether one is consistently faster, whether MM encounters `inf`/`nan` during optimization , etc..., and then override the fit method for all these cases. Is it ok if that's not part of this PR? I was trying to add something general; I didn't intend to consider each of the distributions separately (except where necessary).

Reply:
>Is it ok if that's not part of this PR? I was trying to add something general; I didn't intend to consider each of the distributions separately (except where necessary).

Yeah that's fine, I wasn't asking to include that in this PR, only commenting to determine intended approach when that arises.